app/views/layouts/_user_menu.html.haml

```diff
@@ -16,6 +16,11 @@
     = link_to(conference_program_tracks_path(@conference.short_title)) do
       %span.fa.fa-road
       My Tracks
+-if @conference && current_user.booths.where(conference_id: @conference.id).count > 0
```

Comment:
Since in the rest of the app it is shown no matter the submissions, I could also change it